addons/crm_iap_enrich/models/crm_lead.py

```diff
@@ -79,19 +79,19 @@ def iap_enrich(self, from_cron=False):
                         except iap_tools.InsufficientCreditError:
                             _logger.info('Sent batch %s enrich requests: failed because of credit', len(lead_emails))
                             if not from_cron:
-                                data = {
-                                    'url': self.env['iap.account'].get_credits_url('reveal'),
-                                }
-                                leads[0].message_post_with_view(
-                                    'crm_iap_enrich.mail_message_lead_enrich_no_credit',
-                                    values=data,
-                                    subtype_id=self.env.ref('mail.mt_note').id)
+                                self.env['iap.account']._send_iap_bus_notification(
+                                    service_name='reveal',
+                                    title=_("Not enough credits for Lead Enrichment"),
+                                    is_credit_error=True)
                             # Since there are no credits left, there is no point to process the other batches
                             break
                         except Exception as e:
                             _logger.info('Sent batch %s enrich requests: failed with exception %s', len(lead_emails), e)
```

Comment:
In that case, shouldn't we notify, or do we keep ourselves with credit errors only ?